script/seed.js

```diff
@@ -2,17 +2,118 @@
 
 const db = require('../server/db')
 const {User} = require('../server/db/models')
+const {Product} = require('../server/db/models')
+const {Category} = require('../server/db/models')
+
```

Comment:
Also since the import path is the same all the imports can be combined into one line -

```suggestion
const {Product, Category, ProductCategory} = require('../server/db/models')
```